### ISSUE TITLE
Two Factor setting without effect

### ISSUE THREAD

**Issue by keepxtreme** [andrei-tatar/node-red-contrib-smartnora#27]
I just stumbled accross a bug regarding TwoFactor Authentification: Independent the setting for the Two Factor behaviour the Google Assistant keeps asking me if I’m sure to perfom the action (TwoFactor setting „Acknowledge“). For the same reason using the device in routines or in the Home-App won‘t work, just voice commands.

So far I tested it with the switch-device on 1.0.4 & 1.0.5

**Comment by andrei-tatar** [comment]
Hi, can you please post an example flow?

**Comment by keepxtreme** [comment]
Sure,
The flow you‘re looking for is the one with TV:
`[{"id":"754ba810.54e698","type":"tab","label":"GoogleAssistant","disabled":false,"info":""},{"id":"43124d40.3475b4","type":"inject","z":"754ba810.54e698","name":"","props":[{"p":"payload"},{"p":"topic","vt":"str"}],"repeat":"","crontab":"","once":false,"onceDelay":0.1,"topic":"","payload":"true","payloadType":"bool","x":310,"y":340,"wires":[["1980be26.8b98e2"]]},{"id":"b11bde5d.fc6de","type":"inject","z":"754ba810.54e698","name":"","props":[{"p":"payload"},{"p":"topic","vt":"str"}],"repeat":"","crontab":"","once":false,"onceDelay":0.1,"topic":"","payload":"false","payloadType":"bool","x":310,"y":380,"wires":[["1980be26.8b98e2"]]},{"id":"42ea0e51.56a4b","type":"wake on lan","z":"754ba810.54e698","mac":"C8:08:E9:99:97:7A","host":"192.168.178.255","udpport":9,"name":"turnOn","x":1010,"y":340,"wires":[]},{"id":"a3d18e31.58ca6","type":"switch","z":"754ba810.54e698","name":"","property":"payload","propertyType":"msg","rules":[{"t":"true"},{"t":"false"}],"checkall":"true","repair":false,"outputs":2,"x":670,"y":360,"wires":[["42ea0e51.56a4b"],["84affd86.7b246"]]},{"id":"84affd86.7b246","type":"change","z":"754ba810.54e698","name":"turnOff","rules":[{"t":"set","p":"payload","pt":"msg","to":"turnOff","tot":"str"}],"action":"","property":"","from":"","to":"","reg":false,"x":840,"y":400,"wires":[["ef3874d9.bb5268"]]},{"id":"ef3874d9.bb5268","type":"lgtv-control","z":"754ba810.54e698","tv":"bf372906.b5dfd8","name":"TurnOff","x":1020,"y":400,"wires":[[]]},{"id":"c90f012f.1341d","type":"inject","z":"754ba810.54e698","name":"Start Netflix","props":[{"p":"payload"},{"p":"topic","vt":"str"}],"repeat":"","crontab":"","once":false,"onceDelay":0.1,"topic":"","payload":"true","payloadType":"bool","x":330,"y":620,"wires":[["edeb1695.5cf8b"]]},{"id":"431a1e0c.0942f8","type":"lgtv-app","z":"754ba810.54e698","tv":"bf372906.b5dfd8","name":"Starte App","passthru":true,"x":1010,"y":620,"wires":[["8bb8137.0a3ec7","3dec720b.ad77ce"]]},{"id":"8bb8137.0a3ec7","type":"switch","z":"754ba810.54e698","name":"","property":"payload","propertyType":"msg","rules":[{"t":"neq","v":"app","vt":"flow"}],"checkall":"true","repair":false,"outputs":1,"x":1170,"y":620,"wires":[["8588b3b2.d4bd28"]]},{"id":"c24f3cc4.c2bea8","type":"change","z":"754ba810.54e698","name":"Netflix","rules":[{"t":"set","p":"payload","pt":"msg","to":"app","tot":"flow"}],"action":"","property":"","from":"","to":"","reg":false,"x":850,"y":620,"wires":[["431a1e0c.0942f8"]]},{"id":"edeb1695.5cf8b","type":"change","z":"754ba810.54e698","name":"Setze flow","rules":[{"t":"set","p":"iteration","pt":"flow","to":"1","tot":"num"},{"t":"set","p":"app","pt":"flow","to":"netflix","tot":"str"}],"action":"","property":"","from":"","to":"","reg":false,"x":670,"y":620,"wires":[["c24f3cc4.c2bea8"]]},{"id":"42047f2d.e020c","type":"change","z":"754ba810.54e698","name":"","rules":[{"t":"set","p":"iteration","pt":"flow","to":"flow.iteration+1","tot":"jsonata"}],"action":"","property":"","from":"","to":"","reg":false,"x":1110,"y":700,"wires":[["c24f3cc4.c2bea8"]]},{"id":"8588b3b2.d4bd28","type":"switch","z":"754ba810.54e698","name":"","property":"iteration","propertyType":"flow","rules":[{"t":"lte","v":"3","vt":"num"}],"checkall":"true","repair":false,"outputs":1,"x":1310,"y":620,"wires":[["42047f2d.e020c"]]},{"id":"e264904f.198b8","type":"change","z":"754ba810.54e698","name":"Setze flow","rules":[{"t":"set","p":"iteration","pt":"flow","to":"1","tot":"num"},{"t":"set","p":"app","pt":"flow","to":"lovefilm.de","tot":"str"}],"action":"","property":"","from":"","to":"","reg":false,"x":670,"y":740,"wires":[["c24f3cc4.c2bea8"]]},{"id":"cc761330.a9dcc8","type":"inject","z":"754ba810.54e698","name":"Start Prime","props":[{"p":"payload"},{"p":"topic","vt":"str"}],"repeat":"","crontab":"","once":false,"onceDelay":0.1,"topic":"","payload":"true","payloadType":"bool","x":320,"y":740,"wires":[["e264904f.198b8"]]},{"id":"3dec720b.ad77ce","type":"debug","z":"754ba810.54e698","name":"","active":true,"tosidebar":true,"console":false,"tostatus":false,"complete":"true","targetType":"full","statusVal":"","statusType":"auto","x":1170,"y":560,"wires":[]},{"id":"3ec3fd93.3af272","type":"debug","z":"754ba810.54e698","name":"","active":true,"tosidebar":true,"console":false,"tostatus":false,"complete":"false","statusVal":"","statusType":"auto","x":570,"y":420,"wires":[]},{"id":"1980be26.8b98e2","type":"noraf-switch","z":"754ba810.54e698","devicename":"TV","roomhint":"Wohnzimmer","name":"TV","passthru":true,"nora":"1f6dd59a.bacffa","topic":"","onvalue":"true","onvalueType":"bool","offvalue":"false","offvalueType":"bool","twofactor":"off","twofactorpin":"2809","x":470,"y":360,"wires":[["a3d18e31.58ca6","3ec3fd93.3af272"]]},{"id":"bf372906.b5dfd8","type":"lgtv-config","host":"lgwebostv"},{"id":"1f6dd59a.bacffa","type":"noraf-config","name":"smartNora-SmartTV","group":"Standard","twofactor":"ack","twofactorpin":"","localexecution":true,"structure":"Zuhause"}] `

**Comment by keepxtreme** [comment]
Sorry, my fault, I just found the problem: The global configuration node was set to Acknolage, I changed it to "node" and now it works as expected. However, I would like to make the proposal to change the behaviour as following:
a) in the node, grey out the selection dropdown or show a hint that this setting is overwriten by the configuration node. If not possible, then
b) add the option "inherited" from configuration node and allow nodes to overwrite the configuration nodes setting.
